tests/functional/lambda_func.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=missing-docstring,deprecated-lambda
```

Comment:
https://docs.pylint.org/en/1.6.0/features.html
"This message can’t be emitted when using Python >= 3.0."

Alternatively, we can include a `lambda_func.rc` file with
```
[testoptions]
max_pyver=3.0
```